### PR TITLE
SEO-173296 WPF Chart Label

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @MallikaRK ,

Have renamed image folder, changed image name and image alt.

Regards,
Yvone.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

File: wpf/Charts/Adornments/Label.md [hunk 1]
```diff
@@ -41,12 +41,12 @@ The following code example demonstrates about define the value to be displayed a
 
 | SegmentLabelContent values | Description | Output |
 |---|--|---|
-| DateTime | Displays LabelContent.DateTime value | ![ DateTime in Adornment](Adornments_images/Datetime.png) |
-|LabelContentPath | Without using a LabelTemplate, it displays the y value by default. If we use LabelTemplate for Adornment, we must specify this in to get the underlying data model as DataContext. |![ LabelContentPath in Adornment](Adornments_images/LabelContentPath.png)|
-| Percentage | Displays the percentage value of series' point among other points |![ Percentage in Adornment](Adornments_images/Percentage.png) |
-| XValue | Displays the X value of series' point|![ XValue in Adornment](Adornments_images/Xvalue.png) |
-| YofTot | Displays the value of Y of total values' point|![ YofTot in Adornment](Adornments_images/YofTot.png) |
-| YValue | Displays the Y value of series' point| ![ YValue in Adornment](Adornments_images/YValue.png) |
+| DateTime | Displays LabelContent.DateTime value | ![DateTime in WPF Chart](adornments_images/wpf-chart-datetime.png) |
```
Comment (Girija-Rajendran) [inline wpf/Charts/Adornments/Label.md:44]: why it was removed?
image

Comment (Yvone-Atieno) [reply wpf/Charts/Adornments/Label.md]: have included the removed word

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**